### DISCUSSION TITLE
Does 2.0 support nls?

### DISCUSSION THREAD

**Discussion by wss29** [eclipse-glsp/glsp#1174] [Q&A]
Hi,
    Does glsp 2.0 support nls now?  I want to change to other languages for my application

**Comment by planger** [comment]
Thanks for your message!

As of version 2.0, GLSP doesn't yet support internationalization. We're aware of this need, as highlighted in [#1174](https://github.com/eclipse-glsp/glsp/issues/935), but we haven't had the chance to implement this feature yet.

We're always open to community contributions, so if you're keen to help out, we'd gladly welcome a pull request for this feature. Alternatively, if you're interested in sponsored development to get this feature up and running, we offer this service as well.

Thanks again for your support and interest in GLSP!
